simpegBranchInfo.txt

```diff
@@ -0,0 +1,41 @@
+https://github.com/simpeg/simpeg.git
```

Comment:
Looks like it tagged along with "stuff" on July 8 :) https://github.com/simpeg/simpeg/blob/a298fb420d2866afd5bacec46f7b049eeff07e98/simpegBranchInfo.txt
![image](https://cloud.githubusercontent.com/assets/6361812/18660074/67cc377e-7ec3-11e6-9785-ba64dca595a6.png)